tests/test_interpreter.py

```diff
@@ -140,6 +140,8 @@ def test_print(self):
     def test_factorial(self):
         response = self.get_code_response(FACTORIAL_CODE)
```

Comment:
Can we use the same style as the code below? Something like this
```
self.get_code_response(code=FACTORIAL_CODE, args=TARGET_36_ARG
```

Reply:
Can we also test the behavior of the apps with `args=TARGET_27_ARG` on all of the tests?

Reply:
Done: added `args=TARGET_27_ARG` to all tests except `async_def`